tests/unit/moduleapi/moduleconfigs.tcl

```diff
@@ -0,0 +1,165 @@
+set testmodule [file normalize tests/modules/moduleconfigs.so]
+set testmoduletwo [file normalize tests/modules/moduleconfigstwo.so]
+
+start_server {tags {"modules"}} {
+    r module load $testmodule
+    test {Config get commands work} {
+        # Make sure config get module config works
+        assert_equal [lindex [lindex [r module list] 0] 1] moduleconfigs
+        assert_equal [r config get moduleconfigs.mutable_bool] "moduleconfigs.mutable_bool yes"
+        assert_equal [r config get moduleconfigs.immutable_bool] "moduleconfigs.immutable_bool no"
+        assert_equal [r config get moduleconfigs.memory_numeric] "moduleconfigs.memory_numeric 1024"
+        assert_equal [r config get moduleconfigs.string] "moduleconfigs.string log4j"
```

Comment:
Going to be slightly paranoid and say let's remove "log4j" so nobody asks why we have log4j in our repo.